modules/colors/api/colors-route.ts

```diff
@@ -0,0 +1,62 @@
+import { Request, Response } from 'express';
+import * as core from 'express-serve-static-core';
+
+import getRandomColor from '../utils/getRandomColor';
+import ColorErrors from '../consts/ColorErrors';
+
+const defaultColorSpace = "rgb";
+const defaultColorFormat = "hex";
+
+module.exports = function(app : core.Express){
+
+    app.get("/colors/:qty?", (req: Request, res: Response) => {
+        const qty = req.params.qty ? parseInt(req.params.qty) : 1;
```

Comment:
Hi, Thanks for this awesome work. Do you mind just updating the qty getter to use our util function. getQtyFromRequest

Reply:
Yeah sure